engine/battle/move_effects/counter.asm

```diff
@@ -1,5 +1,4 @@
 BattleCommand_Counter:
-; counter
```

Comment:
I'm going the revert these comments (ex. `; counter`) that I deleted in this PR... However, I feel like they are no longer necessary as we have proper labels now (ex. `BattleCommand_Counter:`). I suggest we create a new issue to start removing some of these from the repo's.

Reply:
Referenced this in #916